src/replica/split/replica_split_manager.cpp

```diff
@@ -781,6 +785,11 @@ void replica_split_manager::on_register_child_on_meta_reply(
 void replica_split_manager::child_partition_active(
     const partition_configuration &config) // on child
 {
+    if (status() != partition_status::PS_PARTITION_SPLIT) {
+        dwarn_replica("child partition has been active, status={}", enum_to_string(status()));
+        return;
+    }
+
     ddebug_replica("child partition become active");
```

Comment:
```suggestion
    if (status() != partition_status::PS_PARTITION_SPLIT) {
        dwarn_replica("child partition has been active, status={}", enum_to_string(status()));
        return;
    }

    ddebug_replica("child partition become active");
```

The two logs seem ambiguous. They both say the partition is "active". I think you should add a comment here, stating what it means "active". And I guess the latter log should be "is becoming active".

Reply:
Yes, I move the latter log after child status changed, it will be clearer.